File: code/e22.m
# part 1
x = linspace(1,100,100);
y = (0.15 * x) + randn(1,100);
plot(x,y,'b.','markersize',15)
title('Noisy Data')
xlabel('X')
ylabel('Y')

# part 2
hold on
z = ((x * 0.05) + 2) + randn(1,100)
plot(x,z,'r.','markersize',15)
ylabel('Y,Z')
legend('Y','Z','location','northwest')

File: code/e22.py
# ipython --pylab

# part 1
x = linspace(1,100,100)
y = (0.15 * x) + randn(100)
plot(x,y,'bo')
title("Noisy Data")
xlabel('X')
ylabel('Y')

# part 2
z = ((x * 0.05) + 2) + randn(100)
plot(x,z,'ro')
legend(("Y","Z"),loc="upper left")
ylabel("Y, Z")
